geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java

```diff
@@ -80,17 +88,18 @@ public void testGeoAdd() {
     memberCoordinateMap.put("Catania", new GeoCoordinate(15.087269, 37.502669));
     Long l = jedis.geoadd("Sicily", memberCoordinateMap);
     assertTrue(l == 2L);
-
-    Region<ByteArrayWrapper, ByteArrayWrapper> sicilyRegion = cache.getRegion("Sicily");
-    assertNotNull("Expected region to be not NULL", sicilyRegion);
-
-    // Check GeoHash
-    String hash =
-        sicilyRegion.get(new ByteArrayWrapper(new String("Palermo").getBytes())).toString();
-    assertEquals("sqc8b49rnyte", hash);
-
-    hash = sicilyRegion.get(new ByteArrayWrapper(new String("Catania").getBytes())).toString();
-    assertEquals("sqdtr74hyu5n", hash);
+    // TODO: make sure GEOADD is tested sufficiently without needing to get the region/Geode
+    // internals and properly implemented
+    // Region<ByteArrayWrapper, ByteArrayWrapper> sicilyRegion = cache.getRegion("Sicily");
+    // assertNotNull("Expected region to be not NULL", sicilyRegion);
```

Comment:
Can we please create a Jira ticket to track the todos?

Reply:
Created  GEODE-7909, updated TODO comments to refer to Jira.